adoc/admin-security-firewall.adoc

```diff
@@ -0,0 +1,5 @@
+= Network Access Considerations
+
+To defend against currently unknown zero-day attacks, it is recommended not to
+expose the kubernetes API server on the public internet. Use network firewalls
+that only allow access from trusted subnets.
```

Comment:
"currently unknown zero-day" is redundant :)

This protects against stolen credentials, prevents credential theft from MITMs, and protects against both known and unknown vulnerabilities.  I don't know if we need to list all the possible reasons, though.  I might just delete everything before the comma, and start with "It is not recommended to expose..."

Reply:
:+1: